server/src/main/java/org/elasticsearch/search/aggregations/metrics/MaxAggregatorFactory.java

```diff
@@ -37,6 +40,23 @@
 
 class MaxAggregatorFactory extends ValuesSourceAggregatorFactory {
 
+    static void registerAggregators(ValuesSourceRegistry valuesSourceRegistry) {
+        valuesSourceRegistry.register(MaxAggregationBuilder.NAME,
+            List.of(CoreValuesSourceType.NUMERIC, CoreValuesSourceType.DATE, CoreValuesSourceType.BOOLEAN),
```

Comment:
Hmm, I wonder if IP should be on this list?  I don't actually know if it works with IP or not, but since NUMERIC == IP in master I suspect it does

Reply:
IP is BYTES in master, not NUMERIC.